templates/general/recent.html

```diff
@@ -44,6 +44,8 @@ <h3 class="box-title"><i class="fa fa-rocket"></i> Recent Scans</h3>
                         <th>App</th>
                         <th>File</th>
                         <th>Type</th>
+                        <th>Package-Name</th>
```

Comment:
More columns make this page look ugly on smaller resolution.
Remove the two new columns, Package Name and Version. 
Move the contents under **App** column

Reply:
Look into `http://127.0.0.1:8000/dynamic_analysis/` for some table styling format that we use.

Reply:
<img width="180" alt="Screenshot 2020-02-27 at 3 35 57 PM" src="https://user-images.githubusercontent.com/4301109/75484486-e159e780-5976-11ea-8181-8c74b7b3133d.png">

A sample here

Reply:
Also in addition to package name, it would be nice to have the app name as well. If you don't mind add that to RecentScansDB

Reply:
Also the two new columns will break **Diff/Compare** feature. If you remove them that will get the issue fixed.